src/langs/en.json

```
@@ -24,9 +24,10 @@
     "CHATBOT_NAME_LABEL": "Chatbot Name",
     "CHATBOT_NAME_HELPER": "This is the name that users will see when they interact with the chatbot",
     "CHATBOT_NAME_DEFAULT_MESSAGE": "by default",
+    "CHATBOT_VERSION_DEFAULT_MESSAGE": "by default",
     "CHATBOT_PROMPT_LABEL": "Chatbot Prompt",
     "CHATBOT_PROMPT_HELPER_LABEL": "In-depth technical description",
-    "CHATBOT_PROMPT_HELPER": "This is the name that users will see when they interact with the chatbot",
+    "CHATBOT_PROMPT_HELPER": "This is a collection of examples of interactions that help the chatbot use an appropriate voice with users.",
```

```suggestion
    "CHATBOT_PROMPT_HELPER": "This is a collection of examples of interactions that define the chatbot's personality.",
``` 
Maybe that's not what you wanted to say, but I think "help" is a bit ambiguous here.